scripts/lint

```diff
@@ -2,6 +2,6 @@
 
 cd "$(dirname "$0")/.."
 
-python3 -m ruff check snitun
-python3 -m ruff format --fix snitun
+python3 -m ruff check --fix snitun
+python3 -m ruff format snitun
```

Comment:
Does `ruff format` exit with error if it reformatted anything?

Reply:
It exits 0 when it reformatted `1 file reformatted, 22 files left unchanged`

Reply:
Since we're using this script in CI I think we want to add the `--check` option.

Reply:
6067b12